pysmt/walkers/identitydag.py

```diff
@@ -73,11 +73,13 @@ def walk_lt(self, formula, args, **kwargs):
         return self.mgr.LT(args[0], args[1])
 
     def walk_forall(self, formula, args, **kwargs):
-        qvars = [self.walk_symbol(v) for v in formula.quantifier_vars()]
```

Comment:
I think this is a matter of preference. I am preparing a commit with the opposite approach.